committee-steering/governance/ug-governance.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes User Group (UG)
+
+## Introduction
+The Kubernetes community and ecosystem benefits providing a place for communication and collaboration between users and community members on topics that either do not have deliverables or are outside the scope of the project itself. These topics are not appropriate for forming SIGs, WGs or Committees, which are intended to drive work within the project. User Groups as the venue for this kind of communication and collaboration.
+
+Example User Groups:
+
+- Big data - Topics related to Big Data integrations.
+- Cloud Providers - Topics related to public/private cloud provider integrations.
+- Machine Learning - Topics related to ML ecosystem integrations.
+- DevOps - Topics related to DevOps tools and deployments.
+
+## UG Goals
+- Facilitate communication between Kubernetes users and contributors on topics pertaining to - the usage of, the extension of and integration with Kubernetes and Kubernetes subprojects.
+
+## UG Non-Goals
```

Comment:
Anti-Goals

Reply:
It was originally Anti-Goals but I had recommended "Non-Goals" in a previous comment since it seemed more consistent with what we use in KEPs and other forms of documents. I'm not strongly held to this opinion though :)

Reply:
I will keep it as Anit Goals, to clear off certain confusions pertaining to UG in the past!